frontend/src/renderer/components/ImageImporter.svelte

```diff
@@ -0,0 +1,161 @@
+<script lang="ts">
+    import { FilePlusIcon } from "svelte-feather-icons";
+    import { processState, sendMessage} from "@util/stores";
+    import { forEach, find } from "lodash";
+    import ImageBubble from "@components/Process/ImageBubble.svelte";
+    import Dropzone from "svelte-file-dropzone";
+    export let label = "Select Files";
+    export let icon = FilePlusIcon;
+    export let largeText = false;
+    export let removeButton = false;
+    let filePaths = [];
+    let files = {
+        accepted: [],
+        rejected: []
+    };
+
+    function getThumbnails() {
+        $processState.thumbnailID = Math.floor(Math.random() * 999999999);
+        let msg = {
+            RequestID: $processState.thumbnailID,
+            RequestType: "Thumbnails",
+            RequestData: {
+                names: filePaths,
+            },
+        };
+        console.log(msg);
```

Comment:
I feel like we could take out some or all of these console.logs

Reply:
I got rid of both of them in this file.